administrator/components/com_templates/src/View/Template/HtmlView.php

```diff
@@ -258,7 +258,7 @@ protected function addToolbar()
 			// Add a copy template button
 			elseif ($this->type === 'home')
 			{
-				$templatesPath = JPATH_SITE . ((int) $this->template->client_id === 1 ? '/administrator' : '') . '/templates/';
+				$templatesPath =  ((int) $this->template->client_id === 1 ? JPATH_ADMINISTRATOR : JPATH_SITE) . '/templates/';
```

Comment:
```suggestion
				$templatesPath = ((int) $this->template->client_id === 1 ? JPATH_ADMINISTRATOR : JPATH_SITE) . '/templates/';
```
No double space, code style.